src/cloud-cli/meltano/cloud/api/client.py

```diff
@@ -287,6 +287,40 @@ async def schedule_put_enabled(
                     raise MeltanoCloudError(ex.response) from ex
                 raise
 
+    async def get_execution_history(
+        self,
+        *,
+        schedule: str | None = None,
+        page_size: int = 10,
```

Comment:
This default value should probably be sourced from a constant `PAGE_SIZE` used both here and within `history.py`.